java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -1542,11 +1542,9 @@ public void setCoordsCenter(Point2D p) {
     }
 
     private void reCalculateCenter() {
-        double centreX = (pointC.getX() - pointA.getX()) / 2;
-        double centreY = (pointC.getY() - pointA.getY()) / 2;
-        centreX = pointA.getX() + centreX;
```

Comment:
@mattharris I wondered that as well, but sat down with a pencil and scrap paper. This change simplifies the full expression with no change in the result. The original code had the algebraic expression (C - A)/2 + A, which simplifies to (C + A)/2, which is expressed in the new code.